### PR TITLE
[MIG] medical_medicament: Migrate to 10.0

### PR DESCRIPTION
**Pull request by kellylougheed**
Depends:
- [x] #119

### CODE REVIEW

**Comment by jcdrubay** [comment]
LGTM

**Comment by lasley** [comment]
Looks like we lost a bit of history in squashing. I went ahead and unshuffled things, rebased onto current 10 branch due to merge, and pushed.

**Review by lasley — APPROVED** [review]
LGTM thanks @kellylougheed

**Comment by lasley** [comment]
WIP pending LasLabs#108, which is the integration of logic from `medical_medicament_component` (#91) into medicament core

Sorry - I admittedly forgot that logic existed, but it's definitely necessary. Hopefully in v11 we remove the remainder of the plain text columns.

**Comment by lasley** [comment]
@kellylougheed - please cherry pick commit from LasLabs#108 now that its approved and merged, then we can get moving on this PR review 😄

**Comment by kellylougheed** [comment]
Done @lasley!

File: medical_medicament/README.rst [hunk 1]
```diff
@@ -6,42 +6,47 @@
...
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
 Contributors
 ------------
 
 * Jonathan Nemry <jonathan.nemry@acsone.eu>
+* Dave Lasley <dave@laslabs.com>
+* Brett Wood <bwood@laslabs.com>
 * Parthiv Patel <parthiv@techreceptives.com>
```
Comment (bmw95) [inline medical_medicament/README.rst]: Remove double contributor entry.

Comment (woodbrettm) [inline medical_medicament/README.rst:46]: Remove double contributor entry

File: medical_medicament/models/product_template.py~Upgrade medical_medicament to v9 & add tests_0 [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (woodbrettm) [inline medical_medicament/models/product_template.py~Upgrade medical_medicament to v9 & add tests_0]: Shrink copyright height

**Review by woodbrettm — APPROVED** [review]
Minor details otherwise LGTM!

**Comment by obulkin** [comment]
The code here is missing certain improvements/fixes that have been made over the course of the year, so I've created a more up-to-date PR: https://github.com/OCA/vertical-medical/pull/209. The current request can now be closed.

**Pull request closed without merging**